src/components/data-management/UploadDetailsModal.jsx

```diff
@@ -99,7 +99,7 @@ const UploadDetailsModal = (props) => {
         }}
         style={{ width: '140px', marginBottom: '10px' }}
       >
-        Replace file
+        {bundleName ? 'Replace file' : 'Upload'}
```

Comment:
You could add a comment explaining that `bundleName` will be undefined if the file hasn't been uploaded